lib/seccomp-tools/cli/cli.rb

```diff
@@ -1,21 +1,30 @@
 # frozen_string_literal: true
 
+require 'os'
```

Comment:
add a blank line after here

(this will be removed per my suggestion below, so this comment is applied when the line is moving to dumper.rb)